src/pretix/base/models/log.py

```diff
@@ -261,3 +202,134 @@ def bulk_postprocess(cls, objects):
         to_wh = [o.id for o in objects if o.webhook_type]
         if to_wh:
             notify_webhooks.apply_async(args=(to_wh,))
+
+
+class LogEntryTypeRegistry(EventPluginRegistry):
```

Comment:
This should, like email placeholders etc, probably get its own module instead of living here with the models, since it's… a lot